syntax = "proto3";

package bayesmix;

/*
 * Enum for the different types of Hierarchy.
 */
enum HierarchyId {
    UNKNOWN_HIERARCHY = 0;
    NNIG = 1; // Normal - Normal Inverse Gamma
    NNW = 2; // Normal - Normal Wishart
    LinRegUni = 3; // Linear Regression (univariate response)
    NNxIG = 4; // Normal - Normal x Inverse Gamma
    LapNIG = 5; // Laplace - Normal Inverse Gamma
    FA = 6; // Factor Analysers
}
